app/controllers/api/v1/notes_controller.rb

```diff
@@ -14,6 +14,11 @@ def show
         render json: note, status: :ok, serializer: NoteDetailSerializer
       end
 
+      def create
+        return handle_invalid_note_type unless valid_note_type?(create_params[:note_type])
```

Comment:
Quizás no sea necesario pasarle un parámetro a esta validación 
```suggestion
        return handle_invalid_note_type unless valid_note_type?
```